1.0.10 type error

## Expected Behavior

Typescript compilation should succeed as it did with 1.0.9.

## Actual Behavior

```
src/components/content/Content.tsx:5:27 - error TS7016: Could not find a declaration file for module 'style-to-object'. 'commafeed-client/node_modules/style-to-object/cjs/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/style-to-object` if it exists or add a new declaration (.d.ts) file containing `declare module 'style-to-object';`

```

## Steps to Reproduce
1. Checkout [this branch](https://github.com/Athou/commafeed/pull/1923)
2. `cd commafeed-client`
3. `npm install`
4. `npm run build`

## Environment

- Version: 1.0.10
- Platform: Any
- Browser: N/A
- OS: Both Linux (CI) and Windows

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -39,3 +39,6 @@ jobs:
 
       - name: Build package
         run: npm run build
+
+      - name: Lint package
+        run: npm run lint:package

--- a/.husky/pre-commit
+++ b/.husky/pre-commit
@@ -1,2 +1,4 @@
 npm run test:ci
+npm run test:esm
+npm run lint:package
 npx lint-staged

--- a/package.json
+++ b/package.json
@@ -5,10 +5,10 @@
   "author": "Mark <mark@remarkablemark.org>",
   "main": "./cjs/index.js",
   "module": "./esm/index.mjs",
-  "types": "./esm/index.d.mts",
+  "types": "./esm/index.d.ts",
   "exports": {
     "import": {
-      "types": "./esm/index.d.mts",
+      "types": "./esm/index.d.ts",
       "default": "./esm/index.mjs"
     },
     "require": {
@@ -24,6 +24,7 @@
     "clean": "rm -rf cjs coverage dist esm",
     "lint": "eslint .",
     "lint:fix": "npm run lint -- --fix",
+    "lint:package": "publint",
     "lint:tsc": "tsc --noEmit",
     "prepare": "husky",
     "prepublishOnly": "run-s lint lint:tsc test clean build",

